db/dbOps.js

```diff
@@ -58,22 +61,36 @@ async function createTables() {
 			ip INET
 		);
 	`);
+	await pool.query(`
+		CREATE TABLE countries(
+			country_code TEXT,
+			country_name TEXT
```

Comment:
could you add here an additional condition:
`unique (country_code)` 
which would help validate the data upon loading the data from the json. It's an extra precaution.

Disregard this comment if we drop this table altogether, so address the previous comment first, to decide whether this one is still relevant.

Reply:
This comment is still valid. If you disagree, kindly ague against it. I want to make sure there is no duplication in the `country_code`. If there is, we will know that there is something wrong with the JSON file.